package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import warnings
+
+from MDAnalysis.analysis.base import AnalysisBase
+
+def dihedral_calc(atomgroups):
```

Comment:
The discussion below converged on eliminating this function and using `lib.distances.calc_dihedrals`.